handlers.go

```diff
@@ -539,12 +542,13 @@ type rpcError struct {
 }
 
 type rpcResult struct {
-	descSource grpcurl.DescriptorSource
-	Headers    []rpcMetadata        `json:"headers"`
-	Error      *rpcError            `json:"error"`
-	Responses  []rpcResponseElement `json:"responses"`
-	Requests   *rpcRequestStats     `json:"requests"`
-	Trailers   []rpcMetadata        `json:"trailers"`
+	descSource   grpcurl.DescriptorSource
+	Headers      []rpcMetadata        `json:"headers"`
+	Error        *rpcError            `json:"error"`
+	emitDefaults bool
+	Responses    []rpcResponseElement `json:"responses"`
+	Requests     *rpcRequestStats     `json:"requests"`
+	Trailers     []rpcMetadata        `json:"trailers"`
```

Comment:
Sorry to nit pick, but would you mind separately the public/private fields while you're in here?  Something like this would be great:
```go
type rpcResult struct {
	Headers      []rpcMetadata        `json:"headers"`
	Error        *rpcError            `json:"error"`
	Responses    []rpcResponseElement `json:"responses"`
	Requests     *rpcRequestStats     `json:"requests"`
	Trailers     []rpcMetadata        `json:"trailers"`

	descSource   grpcurl.DescriptorSource
	emitDefaults bool
}
```
(you'll have to gofmt after)

Reply:
Sure thing! I ran gofmt after to clean it up. The only thing is I am now getting this error which I don't think I modified:

```bash
/home/circleci/repo/cmd/grpcui/grpcui.go:708:55: listener.Addr undefined (type net.Listener has no field or method Addr)
```

I though it may have something to do with the bump of `golang.org/x/net` from v0.17.0 to v0.20.0 but all the checks passed on the dependabot PRs.

Reply:
Ohh... that is super weird.  Doesn't seem like that should be related to your PR :/

Reply:
Looks like it passed on re-run 🤷

Reply:
Awesome! Always nice to have things magically fix themselves. Everything is ready to go on my end then.